03-multiproject/user-feature/table/src/test/java/com/example/myproduct/user/table/TableBuilderTest.java

```diff
@@ -9,6 +9,15 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class TableBuilderTest {
+
+    //TODO (scenario) Running test with TestLauncher API
+    // instructions:
+    //  - goto IDEA registry (Shift, Shift, type "registry")
+    //  - enable "gradle.testLauncherAPI.enabled"
+    //    - remove the line above if the registry value is already set to true in the latest released version of IDEA
+    //  - restart IDEA
+    //  - execute this test
```

Comment:
I think we need a bit more here. The test can potentially work even if the TestLauncher/CC integration is still broken. We'll need two test methods to cover that scenario.
```suggestion
    //  - execute the first test method and verify that it passes and there's a 'Configuration cache entry stored' message in the console log
     //  - execute the second test method and verify that it passes and the configuration cache entry is reused.
    
```